sceptre/scipool/templates/sc-enduser-iam.yaml

```diff
@@ -57,7 +57,18 @@ Resources:
               - organizations:DescribeOrganization
               - servicecatalog:DescribeProvisionedProduct
               - cloudwatch:GetMetricData  # access to EC2 monitoring info
+              - ec2:DescribeImages
             Resource: "*"
+          - Effect: Allow
+            Action:
+              - "ec2:*Tags"
+              - "s3:*Tags"
+              - "*:TagResource"
+              - "*:UntagResource"
+            Resource: "*"
+            Condition:
+              StringEquals:
+                "aws:ResourceTag/aws:servicecatalog:provisioningPrincipalArn": "${aws.PrincipalArn}"
```

Comment:
tags prefixed with `aws:` are [AWS reserved tags and cannot be deleted or updated](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html#tag-restrictions), even by admin users. 
 More info at https://sagebionetworks.jira.com/browse/IT-3504?focusedCommentId=202798

Reply:
I understand that we can't change the value of `aws:servicecatalog:provisioningPrincipalArn`, preventing us from changing ownership of a provisioned product. But we might be able to compare that tag value with the Principal ARN from the request to allow the original product owner to change user tags like `CostCenter` on their own resources.

Reply:
but I'm not entirely sure this will work and I'm not sure how to test it other than merging and seeing what happens